analytics/management/commands/active_user_stats.py

```diff
@@ -40,7 +40,7 @@ def handle(self, *args, **options):
             print("Realm %s" % (realm,))
             for hr, users in sorted(buckets.items()):
                 print("\tUsers for %s: %s" % (hr, len(users)))
-                statsd.gauge("users.active.%s.%shr" %  (statsd_key(realm, True), statsd_key(hr, True)), len(users))
+                statsd.gauge("users.active.%s.%shr" % (statsd_key(realm, True), statsd_key(hr, True)), len(users))
```

Comment:
These are cleanups/lint improvements that I merged into master; they'll disappear from your branch if you do a `git rebase origin/master`.